resources/utilities.nix

```diff
@@ -37,6 +31,14 @@
         -X github.com/cosmos/cosmos-sdk/version.Version=${version}
         -X github.com/cosmos/cosmos-sdk/version.Commit=${src.rev}
         -X github.com/tendermint/tendermint/version.TMCoreSemVer=${tendermint-version}
+        ${additionalLdFlags}
       '';
-    };
+    }
+    // buildGoModuleArgs);
```

Comment:
Would it be more readable if we explicitly inherit the extra arguments here? I have to dig in a bit to find out that the `tags` attribute has been inherited here.

Reply:
It is a common pattern in nix to take an `args` object and just merge it into another attrset 
https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/go-modules/generic/default.nix#L54
https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/go-modules/generic/default.nix#L143

This doesn't make it right, but at least there is precedent for it. I didn't want to enumerate the args because I want to treat it as a kindof pass through were a caller of `mkCosmosGoApp` can override any arg they want to pass to `pkgs.buildGoModule`. This makes the function way more flexible in the future. Its nice to not having to think about the potential args that users would want to override, even collecting the attrs that are overridable is a pain i.e. the only mention of the `subPackages` arg is inline [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/go-modules/generic/default.nix#L197) so even collecting all the attrs to be destructured from `buildGoModuleArgs` would be a pain.

Do you think better documentation of `mkCosmosGoApp` would help here?